src/main/init.js

```diff
@@ -12,14 +12,15 @@
             return this.themes[themeName];
         }
     };
-    
+
     const url = window.location.href.split("/");
-    
+
     const userInfo = {},
           programData = {},
           userPrograms = {};
-    
-    const chromeid = "gniggljddhajnfbkjndcgnomkddfcial";
+
+    //const chromeid = "gniggljddhajnfbkjndcgnomkddfcial";
+    const chromeid = "nemefomcfjaodajlphfgfiomopcekinp";
```

Comment:
Maybe we can all put our local extension IDs here (maybe in array) and access ours when we test it locally.